src/libs/ck-libs/NDMeshStreamer/NDMeshStreamer.h

```diff
@@ -144,21 +223,26 @@ class MeshStreamer : public CBase_MeshStreamer<dtype, RouterType> {
   bool useCompletionDetection_;
   CompletionDetector *detectorLocalObj_;
   virtual int copyDataItemIntoMessage(
-              MeshStreamerMessage<dtype> *destinationBuffer,
-              const void *dataItemHandle, bool copyIndirectly = false);
-  void insertData(const void *dataItemHandle, int destinationPe);
-  void broadcast(const void *dataItemHandle, int dimension,
```

Comment:
Looks like you removed the broadcasting function, does this affect the usage of TRAM in any way? If so it should also be documented.

Reply:
I couldn't find uses of the broadcast function in the code actually. I think it's something that was exclusive to the non-[aggregate] API, so the new API effectively does not support broadcasts.